test/e2e/certs/testca.crt

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Did you set any expiry time?

Reply:
I wonder if we can just create them in run time instead of committing. The reason is that our `creat` script will be unused for 1y or so , so it will most likely no work when we will need it. It might be better to put it in CI and use it.

Reply:
Addressed. It was quite tricky for me but managed to do it after spending hours 😄